src/en/news/blog/2025/rbd-live-migration/index.md

```diff
@@ -0,0 +1,478 @@
+---
+title: "Seamless Data Movement Across Clusters using RBD Live Migration in Squid"
+
+date: 2025-02-07
+author: Sunil Angadi, Anthony D'Atri (IBM)
```

Comment:
Please add `Ilya Dryomov` in the authors he helped me to fix some content in it.